requirements/requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+black
+napari
```

Comment:
@Czaki, for a follow-up, thoughts on removing this if we're handling it via the "extras = " setting?